src/js/unitDivisorSonificationStrategy.js

```diff
@@ -0,0 +1,257 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy", {
+        // can override these to control sonification behaviour
+        // These need better explanations of what they do
+        sonificationOptions: {
+            strategies: {
+                unitDivisor: {
+                    config: {
+                        notes: {
+                            durations: {
+                                play: {
+                                    divisorReturnValue: 3 / 8,
+                                    remainderReturnValue: 1 / 8
+                                }
+                            },
+                            values: {
+                                divisorReturnValue: 91,
+                                remainderReturnValue: 89
+                            }
+                        },
+                        envelope: {
+                            durations: {
+                                play: {
+                                    divisorReturnValue: 1 / 8,
+                                    remainderReturnValue: 1 / 24
+                                },
+                                silence: {
+                                    divisorReturnValue: 1 / 4,
+                                    remainderReturnValue: 1 / 12
+                                }
+                            },
+                            values: {
+                                trueValue: 1.0,
+                                falseValue: 0.0
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        invokers: {
+            "sonifyData": {
+                funcName: "floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy.dataEntriesToSonificationData",
+                args: "{that}"
+            },
+            "unitDivisor10xSonificationStrategy": {
```

Comment:
Is there a need to define 3 strategies by calling the same function with different units? What do you think about defining one invoker that accepts an "unit" argument? This way I think would give more control to users. Also, ```unitDivisor5xSonificationStrategy``` and ```unitDivisor1xSonificationStrategy``` don't seem to be used at all.